src/data/blog/Hiring-web-dev-092019.md

```diff
@@ -26,44 +26,59 @@ author_facebook: null
+We’re recruiting to fill a specific role in our team. If you want to spend time in an office, we've space in Exeter and Norwich, but we're very happy with remote working. We’re also always on the lookout for people who would make our teams stronger, so if this doesn’t suit you please see the information at the end.
 
-##Working mainly in HTML/CSS and Javascript  
+We’re looking for someone who:
 
-You'll have a flair for implementing user interfaces using web technologies. You'll fully understand that what you are designing are interactions, on screens that vary in size, not fixed images on a fixed canvas and enjoy producing interfaces that respond to user needs. Your tools will be HTML, CSS and Javascript and you’ll be passionate about how they work together. If you’re familiar with React, that would be an advantage. You’ll know how to work in a team with several developers and you may already have led some projects or project elements and will be keen to do more team leading.
+- works mainly in HTML/CSS and Javascript
+- has a flair for implementing user interfaces using web technologies. 
+- understands they are developing interactions, on screens that vary in size, that must be accessible to all their users. 
+- is passionate about how  HTML/CSS and Javascript work together, and familiarity with React would be an advantage0
+- can work in a team with other developers, and alongside user researchers and designers to build systems that meet user needs.
 
-##Alongside your core technical skills if you join us you’ll also need to:
+Alongside your core technical skills, you’ll also need to be:
 
-- Be comfortable communicating in a remote team;
-- Enjoy taking part in conversations in technical subjects;
-- Be able to demonstrate that you enjoy expanding your knowledge and thinking critically (this can be via a degree, but 1/3rd of our team members don’t have one, so doesn’t have to be);
-- Value agility in software development (experience of working in an agile environment is an advantage but not essential)
-- Be passionate about accessibility
-- Be keen to work in an organisation that delivers both commercial and social good projects
-- Be inspired by user-centred development (even if you don’t have practical experience of this approach)
-- Be committed to taking part in team activities and individual learning
+- comfortable communicating in a remote team;
+- happy taking part in conversations about technical subjects;
+- able to demonstrate that you enjoy expanding your knowledge and thinking critically (this can be via a degree, but 1/3rd of our team members don’t have one, so doesn’t have to be);
+- value agility in software development (experience of working in an agile environment is an advantage but not essential)
```

Comment:
I'm no fan of semi-colons at the end of list items, but the first three have them and the last four don't.

Reply:
I've made the bullet list punctuation consisten - it was all over the place. Thanks.